demo/scripts/column-chart.js

```diff
@@ -105,19 +105,30 @@ var fixtures = {
         { date: new Date('9/30/05'), value: 1.03},
         { date: new Date('12/30/05'), value:     1.348}
     ],
+    day : [ //this one doesn't work - only graphs 1 day. Not sure we should be using this anyway for days...
```

Comment:
I acknowledge comment - agree not sure how this is related to quarters? If unrelated and also doesn't work but thought to be useful for later?  maybe this should be stored somewhere else? e.g. gist attached to Jira task or a clearly labeled branch.